src/shared/components/common/searchable-select.tsx

```diff
@@ -116,17 +119,14 @@ export class SearchableSelect extends Component<
             ? `${I18NextService.i18n.t("loading")}${loadingEllipses}`
             : options[selectedIndex].label}
         </button>
-        <div
-          role="combobox"
-          aria-activedescendant={options[selectedIndex].label}
```

Comment:
Is active descendent not needed?

Reply:
Ah you're right, it is, I did some brief research on it but got it wrong, this should be there.